examples/ihdp_dataset_dragonnet_vs_metalearners.ipynb

```diff
@@ -0,0 +1,1767 @@
+       "      <td>1.398600</td>\n",
+       "      <td>0.544212</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>S</th>\n",
+       "      <td>3.747818</td>\n",
+       "      <td>1.145911</td>\n",
+       "      <td>0.546522</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>T</th>\n",
+       "      <td>3.818774</td>\n",
+       "      <td>1.094771</td>\n",
+       "      <td>0.558276</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>X</th>\n",
+       "      <td>3.875319</td>\n",
+       "      <td>1.182330</td>\n",
+       "      <td>0.541974</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>actual</th>\n",
+       "      <td>3.844052</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>dragonnet</th>\n",
+       "      <td>3.834915</td>\n",
+       "      <td>1.245932</td>\n",
+       "      <td>0.537838</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"6\" valign=\"top\">ihdp_npci_15.csv</th>\n",
+       "      <th>R</th>\n",
+       "      <td>3.395916</td>\n",
+       "      <td>1.449394</td>\n",
+       "      <td>0.627455</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>S</th>\n",
+       "      <td>4.030133</td>\n",
+       "      <td>1.532327</td>\n",
+       "      <td>0.617098</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>T</th>\n",
+       "      <td>4.060164</td>\n",
+       "      <td>1.180957</td>\n",
+       "      <td>0.640933</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>X</th>\n",
+       "      <td>4.018644</td>\n",
+       "      <td>1.364685</td>\n",
+       "      <td>0.628980</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>actual</th>\n",
+       "      <td>3.929148</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>dragonnet</th>\n",
+       "      <td>4.003075</td>\n",
+       "      <td>1.243092</td>\n",
+       "      <td>0.637877</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"6\" valign=\"top\">ihdp_npci_16.csv</th>\n",
+       "      <th>R</th>\n",
+       "      <td>3.911443</td>\n",
+       "      <td>1.597002</td>\n",
+       "      <td>0.548937</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>S</th>\n",
+       "      <td>4.488440</td>\n",
+       "      <td>1.254707</td>\n",
+       "      <td>0.572632</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>T</th>\n",
+       "      <td>4.698523</td>\n",
+       "      <td>1.132001</td>\n",
+       "      <td>0.580085</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>X</th>\n",
+       "      <td>4.534673</td>\n",
+       "      <td>1.2382
```

Comment:
Is it possible to download the file in the notebook? If so, let's add code to download the IHDP data set. If not, let's gzip and add one of CSV files to the repo and make it available. 

Also let's remove your personal paths in the notebook.